### DISCUSSION TITLE
Numeric Value

### DISCUSSION THREAD

**Discussion by gvhomeassistant** [Blackymas/NSPanel_HA_Blueprint#1093] [Ideas]
Hi, 
I use lots of numeric values for farious purposes, like target temp for my thermostats
Would it be possible in any way to adjust them with NSPannel ? if not possible with a virtual keyboard, maybe possible with + and - or with a slider ? 
Thanks !!!

**Comment by kroessig** [comment]
Maybe you can switch your numeric-item to a lamp or shutter. 
Or you create a new virtual lamp or shutter and copy your value from the lamp or shutter so your numeric value

**Comment by rudgej** [comment]
Have you found a way to display your value? I need to show a time but I cant find a way to display it on the panel ( it is a helper)

**Comment by kroessig** [comment]
I don't know how to do it for a time-value, but for normal values.

--> Lamp or Shutter --> swipe right
![PXL_20231024_165656810 MP](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/139384422/21166fa3-6f97-4b2f-8dbc-4ee2dadd6f34)
--> long press to adjust 
![PXL_20231024_165733391](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/139384422/40f182ff-cbfd-43c5-be3e-a7c723909536)

**Reply by rudgej** [reply]
Thank you - It is time I need to try and display. I can then make it go up or down using buttons - but I need to display it!

**Comment by Ladida1** [comment]
Is there a way to do this?
I´m about to migrate my NSPanels over to this project and I rely heavely on sliders for temperature control.

Temperature presets
Comfort: 20   (Slider for values 15-23 in all cases)
Night: 16
Away: 15
Holiday: 10

**Reply by edwardtfn** [reply]
Are you planning to use one of the relays in the panel to control your heater?

**Reply by Ladida1** [reply]
No, just buttons on the screen.

**Reply by edwardtfn** [reply]
The slider on the panel will follow the limits provided by your integration in this case, which should be based on the preset.
I think it will work fine in your case. 😉

**Comment by edwardtfn** [comment]
- #1732
